src/Triangle.java

```diff
@@ -0,0 +1,67 @@
+
+import java.util.ArrayList;
+
+public class Triangle extends Polygon {
+
+	public Triangle(int numOfSides, ArrayList<Point> coors) { // 2 i
+		super(numOfSides, coors);
+	}
+
+	public Triangle(int numOfSides) { // 2 ii
+		super(numOfSides);
+	}
+
+	public Triangle(ArrayList<Point> coors) { // 3 i
+		super(3, coors);
+
+	}
+
+	
+	public Triangle() { // 3 ii
+		numOfSides = 3;
+	}
+	/*
+	 * public Triangle () {//3 ii super(3); super.addOneCoordinate(2, 3);
```

Comment:
u don't need these many constructors. Keep the 3rd one and modify the 4th one and you would be good.